python/tank/bootstrap/manager.py

```diff
@@ -583,11 +585,41 @@ def get_pipeline_configurations(self, project):
             current_login=self._sg_user.login,
             sg_connection=self._sg_connection,
         ):
+            try:
+                path = ShotgunPath.from_shotgun_dict(pc)
+                config_path = path.current_os
+            except TankError:
+                log.warning(
+                    "Unable to determine config path for %s: %s",
+                    sys.platform,
+                    pc,
+                )
+                cfg_descriptor = None
+            except ValueError:
+                log.warning(
+                    "The PipelineConfiguration %s is not configured for the "
+                    "current platform %s.",
+                    pc,
+                    sys.platform,
+                )
+                cfg_descriptor = None
+            else:
+                try:
+                    cfg_descriptor = create_descriptor(
```

Comment:
Hum, you'll have to fill me in here. I don't understand what is going on. The method states you are returning the descriptor for a config, but are only considering the `*_path` fields with the ShotgunPath and then are casting those to a path descriptor. How does this work with pipeline configurations using descriptors?